core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -207,6 +207,7 @@ private int formatFiles(ArgInfo argInfo, JavaFormatterOptions options) {
     int numThreads = Math.min(MAX_THREADS, argInfo.parameters.fileNamesFlag.size());
     ExecutorService executorService = Executors.newFixedThreadPool(numThreads);
 
+    Map<Path, String> inputs = new LinkedHashMap<>();
```

Comment:
I was imagining something like `FormatResult` as a nested class in `FormatFileCallable`, but this is simple enough that I'm not sure it's worth it. I'll leave it up to you.